bin/init

```diff
@@ -1,5 +1,6 @@
 #!/usr/bin/env bash
 
+# shellcheck source=bin/_config.sh
```

Comment:
Is this required to check `bin/_config.sh` sources?

Reply:
I think yes.